pom.xml

```diff
@@ -1145,6 +1145,35 @@
                 <groups>io.dockstore.common.RegressionTest</groups>
             </properties>
         </profile>
+        <profile>
+            <id>workflow-integration-tests</id>
+            <!-- Only run integration tests for workflows -->
+            <properties>
+                <skipTests>true</skipTests>
+                <skipITs>false</skipITs>
+                <groups>io.dockstore.common.WorkflowTest</groups>
+                <excludeGroups>io.dockstore.common.SlowTest,io.dockstore.common.RegressionTest, io.dockstore.common.ToilOnlyTest</excludeGroups>
+            </properties>
+        </profile>
+        <profile>
+            <id>tool-integration-tests</id>
+            <!-- Only run integration tests for tools -->
+            <properties>
+                <skipTests>true</skipTests>
+                <skipITs>false</skipITs>
+                <groups>io.dockstore.common.ToolTest</groups>
+                <excludeGroups>io.dockstore.common.SlowTest,io.dockstore.common.RegressionTest, io.dockstore.common.ToilOnlyTest</excludeGroups>
+            </properties>
+        </profile>
+        <profile>
+            <id>other-integration-tests</id>
+            <!-- Only run integration tests not related to only tools or workflows. This is a catch all. -->
```

Comment:
I like the catch-all idea. It helps prevent tests from slipping through the cracks which was my concern.